Codigos/Estruturas-de-Dados/XorTrie/XorTrie.cpp

```diff
@@ -0,0 +1,63 @@
+template <int bits = 30> struct XorTrie {
+    vector<vector<int>> go;
```

Comment:
I am in favor of making this `go` static, reversing the order and starting the root
```suggestion
    const int MAXN = 1e6+6;
    int go[MAXN * BITS][2] = {{-1, -1}};
```
this is the pattern is some of our structures, like Seg Trees

Reply:
I prefer `tree` instead of `go`. but it is fine.